ACE/ace/ACE.inl

```diff
@@ -216,7 +216,7 @@ ACE::sendv_n (ACE_HANDLE handle,
 ACE_INLINE ssize_t
 ACE::send_i (ACE_HANDLE handle, const void *buf, size_t len)
 {
-#if defined (ACE_WIN32) || defined (ACE_MQX)
+#if defined (ACE_WIN32) || defined (ACE_MQX) || defined (__ghs__)
```

Comment:
Following convention, shouldn't macros like `__ghs__` and `INTEGRITY` have their own `ACE_*` counterparts?